### DISCUSSION TITLE
Where does the filename come from and can it be changed.

### DISCUSSION THREAD

**Discussion by geertjanknapen1** [spatie/statamic-responsive-images#239] [Q&A] [answered]
So in my control panel I upload an image for each of the breakpoints. As an example;
- test_desktop.jpg
- test_tablet.jpg
- test_mobile.jpg

But when looking at the `<picture>` element and its `srcset`, the images get a, what seems generated, filename. Same thing happens for the underlying `img` element. Is there a way to control that filename?

See below screenshot for an example:
<img width="726" alt="image" src="https://github.com/spatie/statamic-responsive-images/assets/33750300/86549eb8-ac2f-4c57-8680-21114a1ac2cb">

**Comment by ncla** [comment, marked as answer]
Are you wondering this for SEO purposes? Because there was a recent `v4.37.0` update in `statamic/cms` that now appends original file name to the Glide URL paths. Try updating your CMS and see if it helps. Check out this pull request for more info. https://github.com/statamic/cms/pull/8661

**Reply by geertjanknapen1** [reply]
Ah! We are behind on some updates due to some big projects taking precedence. Yes! I was wondering for SEO purposes indeed, but this does seem like it will fix our problem.
Thanks for the quick reply!
